packages/amplify_datastore/example/integration_test/utils/setup_utils.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:amplify_datastore/amplify_datastore.dart';
+import 'package:amplify_datastore_example/amplifyconfiguration.dart';
+import 'package:amplify_datastore_example/models/ModelProvider.dart';
+import 'package:amplify_flutter/amplify.dart';
+
+Future<void> configureAuth() async {
```

Comment:
nit: not auth, could change to datastore or similar

Reply:
Good catch. Copy/paste is not my friend today 😅